src/Illuminate/Foundation/Concerns/ResolvesDumpSource.php

```diff
@@ -37,13 +37,22 @@ public function resolveDumpSource()
             $relativeFile = substr($file, strlen($this->basePath) + 1);
         }
 
+        if (str_starts_with($relativeFile, 'storage/framework/views/')) {
```

Comment:
@baselrabia This cannot be hardcoded like this. The view storage path should be fetched from the config.

https://github.com/laravel/laravel/blob/v9.3.8/config/view.php#L20-L34